core/neat/functions/_neat-merge-defaults.scss

```diff
@@ -0,0 +1,23 @@
+@charset "UTF-8";
+/// Return a map merged with the map of Neat's default settings.
```

Comment:
Thoughts on this describing the _how_ and _what_ behind the purpose of this function, and moving this description down to the description for `@return` on Line 6?